uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerAuthenticatorUpdatedVersionImpl.java

```diff
@@ -45,15 +45,15 @@
 public class BrokerAuthenticatorUpdatedVersionImpl extends BrokerMicrosoftAuthenticator {
 
     private static final String TAG = BrokerAuthenticatorUpdatedVersionImpl.class.getSimpleName();
-    private static final boolean isExpectingMFA = true;
+    public static boolean shouldUseDeviceSettingsPage = true;
```

Comment:
isExpectingMFA flag was initially added because there was a bug in Authenticator app during device registration. It was asking for MFA. If it is resolved now, we can simply remove the flag right?

Reply:
We could, I figured we could keep it for now so as not to mess with any of the other test cases. I don't think the MFA prompt is an issue anymore. I'm hesitant to do that as part of this Pr as many changes are already coming in. I will probably investigate in january maintenance effort